### ISSUE TITLE
Next Steps

### ISSUE THREAD

**Issue by wd7512** [wd7512/rl-health-interventions#296]
Get LLMs to directly output the transition matrix values

- Test across different families of LLMs -> how does the training data change (Mistral, GPT, Deepseek) region based
- Estimate uncertainty from variance of results from multiple exact prompts
- Add a persona and base transition matrix into a prompt to derive how it will change the transition matrix

**Comment by wd7512** [comment]
Find datasets that describes activities throughout a day -> to find the probability distribution of different.

What does this data look like
- Once you have activities you have the probability of moving between states. 
- So for a ground truth we can build transition matrices from this kind of dataset. 

Eventually we put this to the side and use it as a reference point:


We run whatever methodology we have i.e. LLM probability requests and then we compare against this baseline
- check for different features, ages, gender ......

This tells us if our simulator is good or not

**Comment by wd7512** [comment]
1. Prove that our baseline simulation that matches state transitions from a dataset <- foundational contribution
2. Then...... we can have an experimental setup with the actions (which cannot be verified with a dataset) but in an explainable way. <- experimental contribution

**Comment by wd7512** [comment]
Can compare US datasets -> US models -> US persona prompts. This is an interesting experiment contribution. N datasets, M models, P personas/feature of a person. (this is our main initial experimental setup).

Here is a way of modelling the environment for our RL framework. (verified against different groups of the populations against datasets)


Much further down the line, why this reward function is better than another reward function

**Comment by wd7512** [comment]
Does there exist correction factors/functions/prompts that can move us between different transitions matrices that different demographics hold.
